packages/db-sqlite/src/types.ts

```diff
@@ -127,11 +127,11 @@ export type IDType = 'integer' | 'numeric' | 'text'
 
 export type MigrateUpArgs = {
   payload: Payload
-  req?: Partial<PayloadRequest>
+  req?: PayloadRequest
```

Comment:
Why would it be optional if it looks like we do pass it always? (other places as well)
https://github.com/payloadcms/payload/blob/68df4af447547ce9597b02161f070147a836c8e1/packages/drizzle/src/migrate.ts#L78-L87